notification-hubs-sdk/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,5 @@
 <resources>
     <string name="app_name">Notification-hubs-sdk</string>
     <string name="firebase_preference_file_key">com.microsoft.windowsazure.messaging.notificationhubs.FirebasePreferences</string>
+    <string name="preference_file_key">com.microsoft.windowsazure.messaging.notificationhubs.SharedPreferences</string>
```

Comment:
This is the right idea! Personally, I'd name it something that indicates that it's holding installation metadata. Maybe something along the lines of `installation_enrichment_file_key`?